blackbox/docs/sql/reference/drop_ingest_rule.txt

```diff
@@ -0,0 +1,52 @@
+.. highlight:: psql
+
+.. _drop-ingest-rule:
+
+====================
+``DROP INGEST RULE``
+====================
+
+Deletes an existing ingestion rule.
+
+Synopsis
+========
+
+::
+
+    DROP INGEST RULE rule_name
+
+
+Description
+===========
+
+``DROP INGEST RULE`` drops an existing ingestion rule.
+
+Parameters
+==========
+
+:rule_name: The identifier of the rule to drop.
+
+
+Notes
+=====
+.. NOTE::
+
+    When the :ref:`administration_user_management` is enabled, this statement
+    can only be issued by a superuser.
```

Comment:
we should point out, that rules can be dropped even if no plugin for this `source_ident` is available (maybe also add this note on create)